examples/openui5/index.html

```diff
@@ -0,0 +1,20 @@
+<!doctype html>
+<html lang="en" data-framework="openui5">
+
+<head>
+	<meta charset="utf-8">
+	<title>OpenUI5 • TodoMVC</title>
+	<script id="sap-ui-bootstrap" src="https://openui5.hana.ondemand.com/resources/sap-ui-core.js" data-sap-ui-theme="sap_bluecrystal" data-sap-ui-libs="sap.m" data-sap-ui-xx-supportedLanguages="en" data-sap-ui-resourceroots='{"ToDoMVC": "./js"}'></script>
```

Comment:
:+1: extracting the required files from the runtime ZIP and running tests right now.